tests/testDisplay.py

```diff
@@ -183,7 +212,7 @@ def setup_module(module):
 
     parser = argparse.ArgumentParser(description="Run the image display test suite")
 
-    parser.add_argument('backend', type=str, nargs="?", default="virtualDevice",
+    parser.add_argument('backend', type=str, nargs="?", default="ds9",
```

Comment:
I think it has to be the virtual device;  it's the only one that we know is available.